capellambse/model/common/accessors.py

```diff
@@ -1250,6 +1328,9 @@ def create(self, *type_hints: str | None, **kw: t.Any) -> T:
             Initialize the properties of the new object. Depending on
             the object, some attributes may be required.
         """
+        if self.fixed_length and len(self) >= self.fixed_length:
+            raise TypeError("Cannot create elements in a fixed-length list")
```

Comment:
Here again. Imagine having an empty fixed-length `ElementList`. One can still populate up to `fixed_length`-many elements. It is just that this addition would exceed the `fixed_length` constraint.